src/common/components/LiveEvent/index.jsx

```diff
@@ -417,16 +417,18 @@ function LiveEvent({
               flexDirection="column"
               marginLeft="10px"
             >
-              <Text
-                fontSize="md"
-                lineHeight="18px"
-                fontWeight="900"
-                color={textColor}
-                marginBottom="5px"
-                marginTop="0"
-              >
-                {t('live-class')}
-              </Text>
+              {cohorts.length > 0 ? (
```

Comment:
This is not working as a person may have more than one cohort and the live event can be about any of those. You should get the cohort from the live event.